docs/api/core/base/merge_decorators.md

```diff
@@ -0,0 +1,3 @@
+# mirascope.core.base.merge_decorators
+
+::: mirascope.core.base.merge_decorators
```

Comment:
Do we need separate documentation for this?
I've only added API documentation as I couldn't find an appropriate section in the existing documentation.

Reply:
just api is fine I think